### ISSUE TITLE
How can I handle durable.engine.MessageNotHandledException properly?

### ISSUE THREAD

**Issue by maifire** [jruizgit/rules#286]
`from durable.lang import *

def risk7_callback(c):
    print('risk7 fraud detected')

dynamic_ruleset = { 'risk7': {
    'suspect': {
        'run': risk7_callback,
        'all': [
            {'m': {'$gt': {'succ': 30}}}
        ],
    }
}}

get_host().set_rulesets(dynamic_ruleset)

post('risk7', {'succ': 31, 'fail': 11})
post('risk7', {'succ': 30, 'fail': 11})`

Hi, I try your module for checking condition with dynamic rule load json. 
I planned rule file & data from redis and insert it to rule engine and send alert to any other DBs or API.
But when I try above code, in second line, I met below Exception.
_durable.engine.MessageNotHandledException: {'amount': 50}_

I think this will your intended work but don't know how can I handle that properly.
Can you guide it? (when run second line I hope happen nothing or call some other callback)

**Comment by jruizgit** [comment]
Hi, thanks for asking the question. MessageNotHandledException is raised when the object posted doesn't satisfy any rules. In that case, the rules engine does not keep a copy of it. You can can handle the exception and ignore:

```python
try:
    post('risk7', {'succ': 31, 'fail': 11})
    post('risk7', {'succ': 30, 'fail': 11})
except MessageNotHandledException as error:
    pass
```
